insert.md

```diff
@@ -165,38 +165,127 @@ If you don't list column names, the statement will use the columns of the table
 ~~~
 
 ### Insert and Return Values
-
+  
+In this example, the `RETURNING` clause returns the `id` and `balance` of the rows inserted. The language-specific versions assume that you have installed the relevant [client drivers](install-client-drivers.html). 
```

Comment:
@knz, that's not true for one of the two drivers I've included (Python), but I get that it's likely true for most drives. I'll look into how easy it is to add a few more languages and then add that note in.